data/vectorojay.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Vector Ojay",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Photographer, transitioning to Tech || Software Developer(TypeScript, React). Versatile. Inquisitive. Passionate about Designs and UI. Eager for knowledge."
+  "avatar": "https://github.com/vectorojay.png"
```

Comment:
hey @vectorojay there is no need of avatar in new version so you can remove it!

Reply:
Done! Thanks.